src/main/resources/database/schema_lockrepository.sql

```diff
@@ -0,0 +1,87 @@
+BEGIN;
```

Comment:
Why are those .sql files needed?
I think those sql tables are for other functionality from spring-integration which we will not be using.
I believe according to this:
https://docs.spring.io/spring-integration/reference/html/jdbc.html#jdbc-lock-registry
The only table needed is the:

CREATE TABLE INT_LOCK  (
LOCK_KEY CHAR(36),
REGION VARCHAR(100),
CLIENT_ID CHAR(36),
CREATED_DATE TIMESTAMP NOT NULL,
constraint INT_LOCK_PK primary key (LOCK_KEY, REGION)
);